lib/auth/api.go

```diff
@@ -786,6 +787,18 @@ type ReadOktaAccessPoint interface {
 	// GetUsers returns a list of users with the cluster
 	GetUsers(ctx context.Context, withSecrets bool) ([]types.User, error)
 
+	// ListAccessLists returns a paginated list of access lists.
+	ListAccessLists(context.Context, int, string) ([]*accesslist.AccessList, string, error)
+
+	// ListAccessListMembers returns a paginated list of all access list members.
+	// May return a DynamicAccessListError if the requested access list has an
+	// implicit member list and the underlying implementation does not have
+	// enough information to compute the dynamic member list.
+	ListAccessListMembers(ctx context.Context, accessListName string, pageSize int, pageToken string) (members []*accesslist.AccessListMember, nextToken string, err error)
+
+	// GetRoles returns a list of roles.
```

Comment:
Can we just skip comments that don't add any value on top of the self-documenting name of a function? It's just a distraction.

Reply:
+1.

I often find that if I make an effort there is usually something interesting to say, but happy to forgo the comments otherwise.